packages/plugin-lighthouse/src/lib/lighthouse-plugin.integration.test.ts

```diff
@@ -10,3 +10,15 @@ describe('lighthousePlugin', () => {
     expect(pluginConfig.groups).toHaveLength(5);
   });
 });
+
+describe('getRunner', () => {
+  it('should create and execute runner correctly', async () => {
+    // onlyAudits is used to reduce test time
+    const runner = getRunner('https://example.com', {
+      onlyAudits: ['is-on-https'],
+    });
```

Comment:
It is worth noting that there is a log being hidden from the test because of the console mocking: 

![image](https://github.com/code-pushup/cli/assets/40126819/fd0a9c45-1ff1-4033-9f3a-82dc90a596a4)


As to what this log means, I don't know, but since the typing is not actually extracted from LH itself it my mean that its not accepting something there or there is some config error.

Reply:
How can I reproduce it?